### DISCUSSION TITLE
Midterm1_14022

### DISCUSSION THREAD

**Discussion by AmooHashem** [ssc-public/Software-Engineering#6] [Q&A]
می‌توانید در حین آزمون، سوالات خود پیرامون آزمون میان‌ترم اول را در اینجا بپرسید، یا سوالات دیگران را در اینجا مطالعه کنید.

**Comment by hosseinmasoodi** [comment]
عرض سلام و خسته نباشید.

آیا در حل سوالات نیاز به ذکر منبع هست؟

**Reply by alirezababazadeh** [reply]
حوزه پاسخ به سوالات امتحان محدود به کتاب مرجع درس و اسلایدهاست. در صورتی که مطالبی اضافه‌ بر این می‌خواهید بنویسید، مراجع رو ذکر کنید، هرچند تاثیری در نمره نخواهد داشت.
